turbopack/crates/turbopack-ecmascript/src/analyzer/imports.rs

```diff
@@ -562,7 +565,13 @@ pub(crate) fn orig_name(n: &ModuleExportName) -> JsWord {
 
 fn parse_with(with: Option<&ObjectLit>) -> Option<ImportedSymbol> {
     find_turbopack_part_id_in_asserts(with?).map(|v| match v {
-        PartId::Internal(index) => ImportedSymbol::Part(index),
+        PartId::Internal(index, is_for_eval) => {
+            if is_for_eval {
+                ImportedSymbol::PartEvaluation(index)
+            } else {
+                ImportedSymbol::Part(index)
+            }
+        }
```

Comment:
To make it more consistent I think I would change

`PartId::Internal(..., true)` to `PartId::InternalEvaluation(...)` and
`PartId::Internal(...)` to `PartId::Internal(...)`